Develop

kbjude opened the pull request:
Develop branch for the Apple clone project which is read to be merged with all the features implemented. 

Third project ready for submission
No HTML or CSS-Lint errors left

Worked on these files and all content push to develop
Index.html
styles.css
README.md

The resources folder:
These stores all the resources like images and other files that I needed to complete the project

kbjude committed:
W

kbjude committed:
modified css

kbjude committed:
Redesigning the whole challenge

kbjude committed:
added one backgroung

kbjude committed:
Middle sections of images worked on

kbjude committed:
footer work started on

kbjude committed:
Structure ready to be used a few modifications to go

kbjude committed:
Lower footer worked on

kbjude committed:
WIP

kbjude committed:
Page structure complete

kbjude committed:
WIP nav-bar

kbjude committed:
WIP nab-bar

kbjude committed:
WIP nav-bar

kbjude committed:
WIP nav-bar with the apple image in white

kbjude committed:
WIP Main bode content

kbjude committed:
middle block WIP

kbjude committed:
all set now for a PR

kbjude committed:
all commented code removed

kbjude committed:
Middle block ready with out the right font-family

kbjude committed:
gradient added

kbjude committed:
Update .gitignore

kbjude committed:
setting up stickler

kbjude committed:
Create stylelint.config.js

kbjude committed:
Create .stickler.yml

kbjude committed:
Update .gitignore

kbjude committed:
Update README.md

kbjude committed:
Icon changed from font awesome to an image

kbjude committed:
pulled and pushed

nikasulo reviewed (changes requested):
# Under Review by Tolulope Olusakin(TSE)

nikasulo commented:
# Great job with how far you've come @kbjude! :handshake: 
Just a few more steps and you can merge.

#### Horizontal scroll
You have a bit of horizontal scrolling going on in your live version. Likely you have some margin or padding setup that extends beyond the browser's width. Please look into this and make appropriate corrections!

#### No gradient
The title of the project is building with gradients and backgrounds and as such, one of the hard requirements is that your navbar has a gradient as its background color. Please see the requirement below
![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/building_with_backgrounds_and_gradients/navigation_requirement.png)

#### Descriptive Readme
Nice job with your current readme file. Please take a look at this [example]() and improve yours accordingly! See the requirement below.
![screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png)

### Summary
Your project has a lot of potentials to shine. Please look into the following things I outlined above and make appropriate corrections. When all this is done, please submit your project for another review :rocket: 

If you think I missed something, you need some help or you'd just like to chat, please reach out to me on [Twitter](https://twitter.com/oluwadamilareo_) or [instagram](https://instagram.com/oluwadamilare_olusakin)

kbjude committed:
gradient and width adjust

kbjude committed:
Update README.md

kbjude committed:
Update README.md

kbjude committed:
Update README.md

kbjude committed:
Update README.md

kbjude committed:
Update README.md

kbjude committed:
Update README.md

kbjude committed:
Update README.md

kbjude committed:
gradient on nav-bar and page size

kbjude committed:
Updates readme

kbjude commented:
Pull Request 2

**Changes Made:**
The following changes as requested by the TSE : 
- Gradient effected in the nav-bar
- The width and margins adjusted
- Readme.md modified.

**Request**
Review the changes that I was asked to make and grant a green light to go ahead and merge if the changes made to meet the expectation

pelzolga123 commented on index.html:
```diff
@@ -0,0 +1,60 @@
+    <div class="footer-bottom">
+        Copyright © 2014 Apple Inc. All rights reserved. 
+        <div id = "footer-lower"> <a href="#">Terms of Use </a> | <a href="#"> Privacy Policy</a>  </div> 
+    </div>
```
This part should be inside `footer`.

pelzolga123 reviewed (changes requested):
Project requires a few changes.

pelzolga123 commented:
## Hi Jude! ! :christmas_tree: 
### Your project looks :+1: great! Good job with descriptive README file!

Here are few things to correct before project can be approved:
* [ ] Bottom part should contains four element which are links (to another articles or #). You need to add links, right now you have just four images. ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

* [ ] Footer should include links like copyright, terms of use, and privacy policy. Loook at the comment above. ([link to page](https://web.archive.org/web/20140301004610/http://www.apple.com/))

[OPTIONAL]
* It is a good practice to use the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document.

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

kbjude commented:
Hi Olga,
I do not think I have understood the ask in this email that you have just
shared with me. I already have these links in the screenshot. Are there
links that I am missing that I am not aware of. Kindly advise.

Thank you
Jude


On Thu, Jan 2, 2020 at 12:19 AM Olga Pelts <notifications@github.com> wrote:

> Hi Jude! ! 🎄 Your project looks 👍 great! Good job with descriptive
> README file!
>
> Here are few things to correct before project can be approved:
>
>    -
>
>    Bottom part should contains four element which are links (to another
>    articles or #). You need to add links, right now you have just four images.
>    (link to page
>    <https://web.archive.org/web/20140301004610/http://www.apple.com/>)
>    -
>
>    Footer should include links like copyright, terms of use, and privacy
>    policy. Loook at the comment above. (link to page
>    <https://web.archive.org/web/20140301004610/http://www.apple.com/>)
>
> [OPTIONAL]
>
>    - It is a good practice to use the <main> tag
>    <https://www.w3schools.com/tags/tag_main.asp> for the dominant content
>    of the <body> of a document.
>
> Please kindly fix the above issues and send another code review request.
>
> If you have any questions or need help, you can message me on slack (@
> Olga Pelts). *Happy coding*
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kbjude/apple/pull/1?email_source=notifications&email_token=AEBKXYCZ4GFILCVZMTI5XDDQ3UCHFA5CNFSM4KALCAU2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEH5MYFQ#issuecomment-570084374>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEBKXYCSZTBW7ZJDJXKCCMTQ3UCHFANCNFSM4KALCAUQ>
> .
>

pelzolga123 commented:
Ok, the thing is, right now you have four images stored in four `div` tags inside your `footer-up` div. all you need to do is to add `a` tags to each div, so that your images can be links. You can also inspect original website structure with developer tools and see how it is done there.

kbjude committed:
WIP on the image links and main div

kbjude commented:
All changes made as asked by the reviewer:
- added the main tag
- changed the images to links
- the links below the image reviewed and confirmed availability
All set to be merged

kbjude committed:
footer items in a footer tag

nikasulo commented:
# Really beautiful job @kbjude!
You have done a great job incorporating the requested changes, and as such, please merge this and carry on with your level of attention to detail. See you at the very top :rocket:

kbjude committed:
feedback from the issue raised by my team
